stylesheets/_session.scss

```diff
@@ -1,44 +1,99 @@
-$session-color-green: #00F782;
+$session-color-green: #00f782;
 $session-color-white: #fff;
 $session-color-black: #000;
 
-
 $session-font-family: Waza;
 
+$session-transition-duration: 0.25s;
+
+$session-icon-size-sm: 16px;
+$session-icon-size-md: 20px;
+$session-icon-size-lg: 28px;
+
 .session-button {
-    min-width: 165px;
-    width: auto;
-    height: 45px;
-    line-height: 45px;
-    padding: 0 35px 0 35px;
-    font-size: 15px;
-    font-family: $session-font-family;
-    font-weight: 700;
-    cursor: pointer;
-    display: flex;
-    justify-content: center;
-    border-radius: 500px;
-
-    @mixin transparent-background($textAndBorderColor) {
-        background-color: Transparent;
-        background-repeat:no-repeat;
-        overflow: hidden;
-        outline:none;
-        color: $textAndBorderColor;
-        border: 2px solid $textAndBorderColor;
+  min-width: 165px;
+  width: auto;
+  height: 45px;
+  line-height: 45px;
+  padding: 0 35px 0 35px;
+  font-size: 15px;
+  font-family: $session-font-family;
+  font-weight: 700;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  border-radius: 500px;
+
+  @mixin transparent-background($textAndBorderColor) {
+    background-color: Transparent;
+    background-repeat: no-repeat;
+    overflow: hidden;
+    outline: none;
+    color: $textAndBorderColor;
+    border: 2px solid $textAndBorderColor;
+  }
+
+  &.full-green {
+    background-color: $session-color-green;
+    color: $session-color-white;
+  }
+
+  &.green {
+    @include transparent-background($session-color-green);
+  }
+
+  &.white {
+    @include transparent-background($session-color-white);
+  }
+}
+
+.session-icon-button {
+  fill: $session-color-white;
+  opacity: 0.4;
+  cursor: pointer;
+  display: inline-block;
+  transition: opacity $session-transition-duration;
+
+  &.small {
+    line-height: $session-icon-size-sm;
```

Comment:
You could use @mixin to avoid duplicate all thoses values. like I did there: https://github.com/loki-project/loki-messenger/blob/f7fde304abe308b5c80a734e7ff5bfb5948fd9f4/stylesheets/_session.scss#L27